src/style.css

```diff
@@ -0,0 +1,84 @@
+.big {
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  position: relative;
+}
+
+.small {
+  width: 100%;
+  height: 42px;
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-bar {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  background: #fff;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
```

Comment:
move to `html` selector, add fallback fonts
![image](https://user-images.githubusercontent.com/99331793/197976151-82246a7d-1272-4b26-bb9f-e6156efea6b3.png)